ale_linters/ruby/standardrb.vim

```diff
@@ -0,0 +1,23 @@
+" Author: Justin Searls https://github.com/searls, ynonp - https://github.com/ynonp, Eddie Lebow https://github.com/elebow
+" based on the ale rubocop linter
+" Description: StandardRB - Ruby Style Guide, with linter & automatic code fixer
+
+call ale#Set('ruby_standardrb_executable', 'standardrb')
+call ale#Set('ruby_standardrb_options', '')
+
+function! ale_linters#ruby#standardrb#GetCommand(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'ruby_standardrb_executable')
+
+    return ale#handlers#ruby#EscapeExecutable(l:executable, 'standardrb')
+    \   . ' --format json --force-exclusion '
+    \   . ale#Var(a:buffer, 'ruby_standardrb_options')
+    \   . ' --stdin ' . ale#Escape(expand('#' . a:buffer . ':p'))
+endfunction
+
+" standardrb is based on RuboCop so the callback is the same
+call ale#linter#Define('ruby', {
+\   'name': 'standardrb',
+\   'executable_callback': ale#VarFunc('ruby_standardrb_executable'),
+\   'command_callback': 'ale_linters#ruby#standardrb#GetCommand',
+\   'callback': 'ale_linters#ruby#rubocop#Handle',
```

Comment:
Could you move the `Handle` function into a location like `autoload/ale/handlers/ruby.vim` with some other name like `HandleRubocopOutput` and use it both files? I try to avoid having one linter file use a function from another linter file.

Reply:
You got it. Just pushed bd79fadd3aeb78267c0a4304caf2a05d8544e90b